### DISCUSSION TITLE
Dataset and Model Query

### DISCUSSION THREAD

**Discussion by chintu-777** [akdasUAF/ForestFireDetection#28] [Q&A] [answered]
Hi @YaliWang2019 ,

The research paper which I have used  as basepaper for my course research project is known today in Gsoc'24. For my project, I have trained using YOLOv8 and gave better results in terms of accurate object detection. I have followed all discussions and same as I have trained different YOLO models for fire tracking in video for my project. But, YOLOv8 for fire tracking in video gave better results than previous YOLO models.

The GSOC project have proposed for the need for improvement of the deep learning models to accurate detection of fire tracking in videos.  I will propose my findings and find a better model for the project and can be  same draft into proposal. As I have experienced with this project from a 3 months, I am eager to contribute to this project.

**Doubt:-** 
1. For this project are we going to use the IEEE dataset as primary dataset  or Can I utilise the different dataset which I have used for my project for testing different models for videos tracking of fire? 
2. As my findings in research paper, is low-energy consumption is constraint for model selection though it giving higher accuracy. Can I consider ratio of accuracy and low-energy consumption to evaluate the model?

Best regards,
@chintu-777

**Comment by YaliWang2019** [comment, marked as answer]
Hi Vamshi, sorry for the late reply. Thanks for your interest in this project. To answer your questions, first, we use the IEEE FLAME dataset for now, since our own datasets will have a range of differences like angles of taking the fire photos/videos, exposures, background situations, forest densities, and so on, which will result in differences in training models. Second, yeah sure you can consider the ratio of accuracy and low-energy consumption to evaluate the model, it depends on your choices, "low-energy" is not necessarily a condition for this project.
